robot_ws/src/martian_detector/nodes/notifier

```diff
@@ -13,63 +13,66 @@
  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 """
-
+import datetime as dt
 import rospy
 import boto3
 from std_msgs.msg import String
 from std_msgs.msg import Int64MultiArray
 
-aws_region = rospy.get_param('~/notifier/aws_region', 'us-west-2')
-s3_bucket_name = rospy.get_param('~/notifier/s3_bucket_name')
-search_object = rospy.get_param('~/notifier/search_object', 'Alien')
-phone_number = rospy.get_param('~/notifier/phone_number', '')
 
-s3 = boto3.resource('s3', aws_region)
-sns = boto3.client('sns', aws_region)
+DETECTIONS_PREFIX = 'detections'
+# sns = boto3.client('sns', aws_region)  #  Disable SNS for now, may enable it later if needed
 
 class Notifier():
 
     def __init__(self):
-        rospy.loginfo('Initializing Notifier node.')
-        self.object_sub = rospy.Subscriber ('/detected_objects', String, self.notify)
+        rospy.loginfo('(Notifier): initializing')
+        self.object_sub = rospy.Subscriber('/detected_objects', String, self.notify)
         self.pub = rospy.Publisher('/led_cmds', Int64MultiArray, queue_size=1)
 
+        self.aws_region = rospy.get_param('~/notifier/aws_region', 'us-west-2')
```

Comment:
It can get confusing to have multiple defaults defined in nested files, I would suggest setting this default in the launch file. Then, have the application crash and produce an error if launched without a value.

Reply:
It was already there, I just kept it. Looks like the original material was built for us-west-2